node/src/chain_spec.rs

```diff
@@ -661,7 +665,11 @@ fn centrifuge_genesis(
 				.map(|(acc, _)| acc)
 				.collect(),
 			collator_reward: 8_325 * MILLI_CFG,
-			total_reward: 10_048 * CFG,
+			treasury_inflation_rate: Rate::saturating_from_rational(3, 100),
+			last_update: std::time::SystemTime::now()
+				.duration_since(std::time::UNIX_EPOCH)
+				.expect("SystemTime before UNIX EPOCH!")
+				.as_secs(),
```

Comment:
Yes, thats the implication. However, this only affects new networks, i.e. either Development if it needs to be reset or otherwise local ones. Therefore, it shouldn't have an impact. If we make it static, the first treasury inflation reward will huge.

Reply:
But you're using the system clock, not the clock used on-chain (that changed each block by 12 secs). I guess we're okay with it.

Reply:
Good point but it should be fine here. It only affects the first treasury reward and should never affect live chains/production.